src/components/HelpDrawer/HelpContent/CatalogOverlayHelpComponent.tsx

```diff
@@ -2,19 +2,18 @@ import * as React from "react";
 import {ImageComponent} from "./ImageComponent";
 import headCatalogueButton from "static/help/head_catalogue_button.png";
 import headCatalogueButton_d from "static/help/head_catalogue_button_d.png";
+import demoCatalogueMarkerMapping from "static/help/demo_catalog_marker_mapping.png";
+import demoCatalogueMarkerMapping_d from "static/help/demo_catalog_marker_mapping_d.png";
 
 export class CatalogOverlayHelpComponent extends React.Component {
     public render() {
         return (
             <div>
                 <p><ImageComponent light={headCatalogueButton} dark={headCatalogueButton_d} width="90%"/></p>
-                <p>Source catalogue files in the VOTable or FITS format can be loaded in CARTA (via &quot;File&quot;-&gt; &quot;Import catalog&quot;) for visualization as an image overlay, or a 2D scatter plot, or a histogram.</p>
+                <p>Source catalogue files in the VOTable or the FITS format can be loaded in CARTA (via &quot;File&quot;-&gt; &quot;Import catalog&quot;) for visualization as an image overlay, or a 2D scatter plot, or a histogram.</p>
```

Comment:
Change to `Source catalogue files in VOTable or FITS format`

Reply:
fixed thanks.